python/delta_sharing/delta_sharing.py

```diff
@@ -96,6 +98,36 @@ def load_as_spark(url: str) -> "PySparkDataFrame":  # noqa: F821
     return spark.read.format("deltaSharing").load(url)
 
 
+def load_table_changes_as_pandas(
+    url: str,
+    starting_version: Optional[int] = None,
+    ending_version: Optional[int] = None,
+    starting_timestamp: Optional[str] = None,
+    ending_timestamp: Optional[str] = None
+) -> pd.DataFrame:
+    """
+    Load the table changes of shared table as a pandas DataFrame using the give url.
+
+    :param url: a url under the format "<profile>#<share>.<schema>.<table>"
+    :param starting_version: The starting version of table changes
+    :param ending_version: The ending version of table changes
+    :param starting_timestamp: The starting timestamp of table changes
+    :param ending_timestamp: The ending timestamp of table changes
+    :return: A pandas DataFrame representing the shared table.
```

Comment:
Similar comment as the other, since this is public facing, we should perhaps add some comments on valid combinations.

Reply:
Added the comment.